commit 0b873aab346559416d37573672f49141d345ff02
Author: Kelvin Zhang <zhangxp1998@gmail.com>
Date:   Fri Dec 31 11:57:52 2021 -0800

    Fix gir scan linker error
    
    Need to link gdk-pixbuf as a dependency, otherwise build for static
    library will fail due to missing libpng/libgjpeg

diff --git a/gdk-pixbuf/meson.build b/gdk-pixbuf/meson.build
index 8b0590b..45546ad 100644
--- a/gdk-pixbuf/meson.build
+++ b/gdk-pixbuf/meson.build
@@ -241,6 +241,7 @@ if build_gir
     identifier_prefix: 'Gdk',
     symbol_prefix: 'gdk',
     export_packages: 'gdk-pixbuf-2.0',
+    dependencies: loaders_deps,
     includes: [ 'GModule-2.0', 'Gio-2.0', ],
     header: 'gdk-pixbuf/gdk-pixbuf.h',
     install: true,
@@ -255,6 +256,7 @@ if build_gir
     identifier_prefix: 'Gdk',
     symbol_prefix: 'gdk',
     export_packages: 'gdk-pixbuf-2.0',
+    dependencies: loaders_deps,
     includes: [ gdkpixbuf_gir[0] ],
     header: 'gdk-pixbuf/gdk-pixdata.h',
     install: true,
@@ -338,7 +340,7 @@ foreach bin: gdkpixbuf_bin
 
   bin = executable(bin_name, bin_source,
                    gdkpixbuf_enum_h,
-                   dependencies: gdk_pixbuf_deps + [ gdkpixbuf_dep ],
+                   dependencies: [ gdkpixbuf_dep ],
                    include_directories: [ root_inc, gdk_pixbuf_inc ],
                    c_args: common_cflags + gdk_pixbuf_cflags,
                    install: true)
